resources/assets/uad_lists.json

```
@@ -14924,7 +14924,7 @@
   {
     "id": "tv.peel.app",
     "list": "Misc",
-    "description": "Peel Universal Smart TV Remote Control (discontinued)\nLets you remotely control devices like your TV, DVD or Blu-ray player.\n",
+    "description": "Peel Universal Smart TV Remote Control (discontinued)\nLet's you remotely control devices like your TV, DVD or Blu-ray player.\n",
```

```suggestion
    "description": "Peel Universal Smart TV Remote Control (discontinued)\nLets you remotely control devices like your TV, DVD or Blu-ray player.\n",
```
Incorrect. "Lets" is correct: https://www.grammarly.com/blog/lets-vs-lets/#:~:text=The%20difference%20between%20let's%20and%20lets&text=Let's%20is%20a%20contraction%20of%20%E2%80%9Clet%20us.%E2%80%9D%20You%20use,to%20allow%20or%20give%20permission.